standard-deployment.cfg

```diff
@@ -18,6 +19,7 @@ parts -=
 
 usernamelogger_ac_cookie_name = __ac
 raven_project_dist = opengever.core
+slack-webhook = $STANDARD_SLACK_WEBHOOK
```

Comment:
I don't exactly understand what context this `$VARIABLE` is supposed to work in - from what I understand, this will just set the buildout var `${buildout:slack-webhook}` to the literal string `"$STANDARD_SLACK_WEBHOOK"`, but not read the shell environment variable `$STANDARD_SLACK_WEBHOOK`. Or I'm misunderstanding what's being done here 😉